tests/snapshots/tests.contrib.asyncpg.test_asyncpg.test_select.json

```diff
@@ -0,0 +1,53 @@
+[[
+  {
+    "name": "postgres.connect",
+    "service": "postgres",
+    "resource": "postgres.connect",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "sql",
+    "meta": {
+      "db.name": "postgres",
+      "db.user": "postgres",
+      "out.host": "127.0.0.1",
+      "runtime-id": "7b513eaf1f8d43c69d123f85db6a01b4"
+    },
+    "metrics": {
+      "_dd.agent_psr": 1.0,
+      "_dd.top_level": 1,
+      "_dd.tracer_kr": 1.0,
+      "_sampling_priority_v1": 1,
+      "out.port": 5432,
+      "system.pid": 25785
+    },
+    "duration": 7144000,
+    "start": 1646899182554664000
+  }],
+[
+  {
+    "name": "postgres.query",
+    "service": "postgres",
+    "resource": "SELECT 1",
+    "trace_id": 1,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "sql",
+    "meta": {
+      "db.name": "postgres",
+      "db.user": "postgres",
```

Comment:
do we want `sql.rows` as well?

do we get that info?

Reply:
So it doesn't look like it's provided in the API and I don't think we'd want to try to call `len` on any results in case they are iterators and we end up doing something expensive.

https://cs.github.com/MagicStack/asyncpg?q=rowcount